docs/cloud-native-security/index.rst

```diff
@@ -149,6 +149,8 @@ many findings, we can reduce the minimum severity level and start responding
 to findings with a lower severity. Likewise, if the current setting is
 generating too many findings, we can increase the minimum severity level.
 
+From time to time, customers might want Rackspace to investigate a finding that is below the defined minimum severity level. Such functionality is available directly in the Security Hub user interface of the AWS console on the master account. By selecting a finding and choosing "Send to Rackspace" within the **Actions** drop-down menu, the finding will be sent to the Rackspace SIEM and investigated by a specialist. Standard charges will apply for findings sent to Rackspace using this method. In order to protect customers from unintended over-charges, the "Send to Rackspace" action supports single findings only. In order to send multiple findings to Rackspace, customers should repeat the process for each finding individually.
+
```

Comment:
"Send to Rackspace"  is for investigation *and* remediation where the customer qualifies. That may be implicit in this text, but we should be explicit where there is any potential for misunderstanding. Otherwise, this looks good.